RN v13.0.0 for Preview (#1372)
mia-platform/documentation: docs/release-notes/v13.0.0.md

image: "img/release-note-link-preview.png"
---

_May 09th, 2024_

> **Welcome to Mia-Platform v13!**

Reviewer: ```suggestion
:::info
Mia-Platform Console v13.0.0 is **now in Preview!** and will be generally available on May 30th.

Console SaaS users can try out v13.0.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.

For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
:::

**Welcome to Mia-Platform v13!**
```

:::

From anywhere on the Console, you can interact with Mia-Assistant, an AI chatbot that will help you simplify your experience when using the Console. 
You can ask Mia-Assistant any question related to the Mia-Platform context: it will answer and provide the related Mia-Platform documentation reference, to help you retrieve all the information you need!

You only need to simply click on the new "Ask Mia-Assistant" button and try it out!

![Mia-Assistant](img/mia-assistant.png)

### Blueprint and Template management via User Interface

We're excited to unveil a significant enhancement to our platform: Blueprint and Template management via the User Interface. This update transforms how you handle your company's Blueprint alongside their respective Templates and settings.

With our new user-friendly interface, managing your Blueprint and Templates becomes a breeze. Say goodbye to the complexity of the CMS interface and hello to streamlined setup. This new section can simplify your project processes, gain better control over them, and seamless user experience.

![Project Blueprint](img/project-blueprint.png)

> To learn more about Blueprint, please refer to the related [documentation page](/docs/development_suite/company/project-blueprint.md).

Reviewer: ```suggestion
To learn more about Blueprint, please refer to the related [documentation page](/development_suite/company/project-blueprint.md).
```

#### New Flow Manager Configurator features

From now on, the Flow Manager Configurator provides the ability to search for Flow Manager services within the project and to search for states and events of the flow currently being viewed which are selected and centered within the canvas.

#### Get user roles on all Company resources 

The API to retrieve user roles on a Company now includes in the response all the resources the user has access to based on hierarchical permissions (even without a direct binding associated with it)

#### Revision/Version info available in the payload sent by the Orchestrator Generator

From now on, also the name of the Revision/Version is included in the payload sent by the Orchestrator Generator.

### Bug Fix

This version addressed the following bugs:

* the `Flow Manager Configurator` button in the microservices section correctly redirects to the right flow
* in the Flow Manager Configurator, the `isFinal` flag is automatically disabled when events are edited via drag and drop

## Fast Data

### Fast Data Control Plane

Mia-Platform Fast Data has been empowered with a **Runtime Management solution**, let's welcome **Fast Data Control Plane**.

> ![Fast Data Control Plane Frontend](img/fd-single-view-pipeline.png)
> 
> From Marketplace, create, configure and deploy [Fast Data Control Plane](/runtime_suite_applications/fast-data-control-plane/10_overview.md) application to start monitoring and governing your Fast Data Runtime!
> This Fast Data suite component provides a general overview of the Fast Data pipelines released on the runtime environment.
> In addition, from its user-friendly web interface, users can _pause_ and _resume_ the consumption of data streams along **Fast Data pipelines**
> with just a few clicks to strategically allocate resources consumption and efficiently utilize them during peak demand processes.

Reviewer: ```suggestion
From Marketplace, create, configure and deploy [Fast Data Control Plane](/runtime_suite_applications/fast-data-control-plane/10_overview.md) application to start monitoring and governing your Fast Data Runtime!
This Fast Data suite component provides a general overview of the Fast Data pipelines released on the runtime environment.
In addition, from its user-friendly web interface, users can _pause_ and _resume_ the consumption of data streams along **Fast Data pipelines**
with just a few clicks to strategically allocate resources consumption and efficiently utilize them during peak demand processes.

![Fast Data Control Plane Frontend](img/fd-single-view-pipeline.png)
```

  
It is **breaking** on targets of type `mia-console`, which now requires private keys to be an object with the following format

  ```json
  {
    "type": "file",
    "path": "/path/to/key"
  }
  ```
instead of a plain path string.

##### New Features

[Salesforce SObjects](/runtime_suite_templates/data-catalog/20_configuration.md#salesforce-sobjects) have been added among the supported Data Sources, so that it is possible to configure the Agent to gather metadata from the ones deployed in your projects environments.

##### Improvements

* introduce support for insecure tls connections on all http clients
* introduce support for custom root certificates or self-signed certificates on all http clients

:::info
In order to get access to this new feature, please remember to rebuild your Data Catalog Agent image against the newer Agent template version.
:::

## Microfrontend Composer

### Configurator

#### Bug Fix

This version addressed the following bugs:

* in the Microfrontend Composer it is now possible to generate the data schema for MongoDB Views, Projections and Single Views.

### Back-Kit Library

The new version `v1.5.1` of the back-kit library is available!

> Refer to back-kit [changelog](https://docs.mia-platform.eu/docs/microfrontend-composer/back-kit/changelog) for finer-grained details of new versions.

Reviewer: ```suggestion
Refer to back-kit [changelog](/microfrontend-composer/back-kit/changelog.md) for finer-grained details of new versions.
```

image: "img/release-note-link-preview.png"
---

_May 09th, 2024_

> **Welcome to Mia-Platform v13!**

Reviewer 2: remember to put the preview banner

Reviewer: inserted
